src/molecules/formfields/util/BaseDateField/index.js

```diff
@@ -112,36 +112,39 @@ class BaseDateField extends React.Component {
   isValidMonthInput(monthValue) {
     const monthString = String(monthValue);
 
-
     return !monthString || inRange(+monthValue, 0, 13);
   }
 
   isValidDayInput(dayValue) {
+    const { monthValue } = this.state;
+
     const dayString = String(dayValue);
-    const maxDate = get(LAST_DATE_MONTH, `${Number(this.state.monthValue)}`, 32);
+    const maxDate = get(LAST_DATE_MONTH, `${Number(monthValue)}`, 32);
 
     return !dayString || inRange(+dayValue, 0, maxDate);
   }
 
-  set month(monthValue) {
+  set month(value) {
```

Comment:
shouldn't this be `monthValue` still?

Reply:
had to make this change because of `react/destructuring-assignment` - had to pull `monthValue` off of `state` so I had to change the parameter name